src/test/java/emissary/config/ServiceConfigGuideTest.java

```diff
@@ -305,13 +305,14 @@ void testEntryPrimitives() throws IOException {
                         + "BOOLEANQ = \"BOGUS\"\n" + "SZP = \"123\"\n" + "SZB = \"123b\"\n" + "SZK = \"123k\"\n" + "SZM = \"123m\"\n"
                         + "SZG = \"123g\"\n" + "SZT = \"123t\"\n" + "SZQ = \"red balloons\"\n" + "SZ1 = \"111\"\n" + "SZ2 = \"222\"\n"
                         + "SZ3 = \"333\"\n" + "SZ4 = \"444\"\n" + "SZ5 = \"555\"\n" + "SZ6 = \"666\"\n" + "SZ7 = \"777\"\n" + "SZ8 = \"888\"\n"
-                        + "SZ9 = \"999\"\n" + "SZ0 = \"000\"\n" + "STRING = \"chars\"\n").getBytes();
+                        + "SZ9 = \"999\"\n" + "SZ0 = \"000\"\n" + "STRING = \"chars\"\n" + "LONG2 = 12345678901234").getBytes();
```

Comment:
Just for awareness:  prior to this PR's change in ServiceConfgGuide.java, the only way to configure a Long value greater than Integer.MAX_VALUE was to surround the value with quotes.  While that technique/workaround worked, it wasn't documented and certainly wasn't something that people would consciously do.